pandas/io/parsers/readers.py

```diff
@@ -325,9 +325,29 @@
     default cause an exception to be raised, and no DataFrame will be returned.
     If False, then these "bad lines" will be dropped from the DataFrame that is
     returned.
+
+    .. deprecated:: 1.3
+       The ``on_bad_lines`` parameter takes precedence over this parameter
+       when specified and should be used instead to specify behavior upon
+       encountering a bad line instead.
 warn_bad_lines : bool, default True
     If error_bad_lines is False, and warn_bad_lines is True, a warning for each
     "bad line" will be output.
+
+    .. deprecated:: 1.3
```

Comment:
might need a blank line *after* the deprecated, not sure how these will render

Reply:
Renders fine. Just needs space before I think. 
![image](https://user-images.githubusercontent.com/47963215/113519897-16eede00-9544-11eb-87b2-cc55fd529cde.png)
(From doc build artifact)